File: palindrome-number/palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if(x<0)
            return false;
        
        long int reversedNum = 0;
        int originalNum = x;
        while(x!=0)
        {
            int temp=x%10;
            x/=10;
            reversedNum = reversedNum*10 + temp;
        }
       
        if(reversedNum == originalNum)
            return true;
        return false;
    }
};

File: palindrome-number/palindrome-number.js
function isPalindrome(x)
	{
		var rem, temp, final = 0;
		var number = x

		temp = number;
		while(number>0)
		{
			rem = number%10;
			number = parseInt(number/10);
			final = final*10+rem;
		}
		if(final==temp)
		{
			return true
		}
		else
		{
			return false
		}
  }
